pointmatcher/DataPointsFilters/OctreeGrid.cpp

```diff
@@ -171,9 +174,11 @@ bool OctreeGridDataPointsFilter<T>::CentroidSampler::operator()(Octree<T>& oc)
 		//retrieve index from lookup table if sampling in already switched element
 		if(std::size_t(d)<idx)
 			j = mapidx[d];
-			
+		
+		T acc[featDim-1] = {0.};
```

Comment:
Jenkins is not happy because of :
```
/home/jenkins/workspace/libpointmatcher/pointmatcher/DataPointsFilters/OctreeGrid.cpp:178:25: error: variable-sized object 'acc' may not be initialized
   T acc[featDim-1] = {0.};
                         ^
```
Btw. variable length arrays such as this one never made it into the c++ standard (AFAIK). It exists as compiler specific extensions and as part of c99. I recommend using `std::vector`. I know, it is sad to accept the additional malloc and free, but compared to the cost of the rest of the function it does not contribute significantly.